lib/matplotlib/backends/backend_nbagg.py

```diff
@@ -98,6 +103,7 @@ def connection_info():
     'zoom_to_rect': 'fa fa-square-o icon-check-empty',
     'move': 'fa fa-arrows icon-move',
     'download': 'fa fa-floppy-o icon-save',
+    'export': 'fa fa-file-picture-o icon-picture',
```

Comment:
Can you elaborate on what "export" does?  It inserts a static image into the notebook?